Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -19,7 +19,7 @@ namespace Emby.Server.Implementations.HttpServer
     /// <summary>
     /// Class WebSocketConnection.
     /// </summary>
-    public class WebSocketConnection : IWebSocketConnection, IDisposable
+    public class WebSocketConnection : IWebSocketConnection, IAsyncDisposable, IDisposable
```

Comment:
You should put `IAsyncDisposable, IDisposable` on the `IWebSocketConnection` interface

Reply:
looks like it was already present

Reply:
looks like it was already present

Reply:
I overlooked that you were talking about the interface... My bad.